code/datums/elements/footstep.dm

```diff
@@ -117,10 +117,10 @@
 		return
 	playsound(source_loc, pick(footstep_sounds[turf_footstep][1]), footstep_sounds[turf_footstep][2] * volume, TRUE, footstep_sounds[turf_footstep][3] + e_range, falloff_distance = 1, vary = sound_vary)
 
-/datum/element/footstep/proc/play_humanstep(mob/living/carbon/human/source, atom/oldloc, direction)
+/datum/element/footstep/proc/play_humanstep(mob/living/carbon/human/source, atom/oldloc, direction, forced, list/old_locs, momentum_change)
 	SIGNAL_HANDLER
 
-	if (SHOULD_DISABLE_FOOTSTEPS(source))
+	if (SHOULD_DISABLE_FOOTSTEPS(source) || !momentum_change)
```

Comment:
approximation of "you didnt actually move with your legs since something else moved you, so dont make footstep sounds"

Reply:
should comment this